cns/restserver/restserver.go

```diff
@@ -72,6 +72,7 @@ type HTTPRestService struct {
 	cniConflistGenerator       CNIConflistGenerator
 	generateCNIConflistOnce    sync.Once
 	IPConfigsHandlerMiddleware cns.IPConfigsHandlerMiddleware
+	PnpIDByMacAddress          map[string]string
```

Comment:
When will this get deleted?

Reply:
it's an in memory cache, it gets deleted when cns terminates.